modules/apps/29-fee/keeper/escrow.go

```diff
@@ -38,6 +39,17 @@ func (k Keeper) EscrowPacketFee(ctx sdk.Context, identifiedFee types.IdentifiedP
 
 	// Store fee in state for reference later
 	k.SetFeeInEscrow(ctx, identifiedFee)
+
+	// Emit event so that relayers know an incentivized packet is ready to be relayed
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeSendIncentivizedPacket,
+			sdk.NewAttribute(channeltypes.AttributeKeyPortID, identifiedFee.PacketId.PortId),
```

Comment:
Should maybe put the relayer address here as well?

Reply:
Sounds like a good idea.
Should we add an `events.go` and move the event emissions funcs there? [Similarly to 04-channel](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/keeper/events.go)

Reply:
I actually think its unnecessary. Right now, relayer list is guaranteed to be nil, so no point in emitting it.

And once it does become non-nil, we can add it to events in a non-breaking manner

Reply:
> we can add it to events in a non-breaking manner

this won't necessarily be true by the time the change is made (as it does seem there is desire to eventually include events in consensus), but I agree we can leave it off for now